spec.bs

```diff
@@ -2863,8 +2902,10 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
   Insert these steps immediately after step 20, the step that goes [=in parallel=], so that what
   follows are the first steps that run [=in parallel=] in the patched algorithm:
 
-    1. If |url| is a [=urn uuid=] and |navigable| is a [=fenced navigable container/fenced
-       navigable=]:
+    1. If |url| is a [=urn uuid=], |navigable| is a [=fenced navigable container/fenced navigable=],
+       and <var ignore>sourceDocument</var>'s [=node navigable=] is in |navigable|'s
+       [=navigable/active document=]'s [=Document/ancestor navigables=] with [=an-unfenced|
+       unfenced=] set to true:
```

Comment:
How about this: instead of adding this condition to patch up the concern found in #194, let's leave it open to the bugs described there, and briefly describe why this step is not sufficient and that #194 tracks this, in an `Issue:` indented below this?